packages/medusa/src/services/product-category.ts

```diff
@@ -86,7 +86,8 @@ class ProductCategoryService extends TransactionBaseService {
    */
   async retrieve(
     productCategoryId: string,
-    config: FindConfig<ProductCategory> = {}
+    config: FindConfig<ProductCategory> = {},
+    selector: Selector<ProductCategory> = {},
```

Comment:
suggestion: I would say selector and config argument. here, you can pass an id, and in the selector another id

Reply:
you can look at the retrieve_ method from the product service 👍

Reply:
That really doesn't look clean, it feels like we're jamming everything into config.

Reply:
it is in the selector not in the config

Reply:
Resolved on call.